src/chunk_size/rust_tokenizers.rs

```diff
@@ -0,0 +1,52 @@
+use rust_tokenizers::{
+    tokenizer::{BertTokenizer, Tokenizer, TruncationStrategy},
+    Offset,
+};
+use std::ops::Range;
+
+use crate::{ChunkCapacity, ChunkSize, ChunkSizer};
+
+impl ChunkSizer for &BertTokenizer {
```

Comment:
I feel like we should be able to implement this for anything that implements `Tokenizer`...

Maybe something like:
```rust
impl<T> ChunkSizer for &T where T: Tokenizer {
```
(I didn't try it out... just did it from memory, hopefully it will still compile...)

Reply:
I tried to follow this path but I feel that the compiler guides me in wrong direction. 
I did get to a point when it complains that it conflicts with `tokenizers` - the other feature that implements ChunkSizer

Reply:
Ah shoot I forgot there are issues implementing a trait for another trait from another package... Might have to implement a macro to implement it for all of their structs or something. 

But I think it would make sense to just get this Bert tokenizer merged in first